app/serializers/page_serializer/groups_show_page_serializer.rb

```diff
@@ -54,24 +55,24 @@ def section_literals
 
     def literals
       [].tap do |items|
-        items << { 'term': { 'content': 'Name' }, 'description': [{ 'content': @group.groupName }] }
+        items << { 'term': { 'content': 'Name' }, 'description': [{ 'content': @group.groupName }] } if @group.try(:groupName)
         items << { 'term': { 'content': 'Start Date' }, 'description': [{ 'content': l(@group.start_date) }] } if @group.try(:groupStartDate)
         items << { 'term': { 'content': 'End Date' }, 'description': [{ 'content': l(@group.end_date) }] } if @group.try(:groupEndDate)
       end.compact
     end
 
     def objects
       @layings.map do |laying|
-        ComponentSerializer::CardComponentSerializer.new(name: 'card__generic', data: { card_type: 'small', heading: card_heading(laying), paragraph: card_paragraph(laying) }).to_h
+        ComponentSerializer::CardComponentSerializer.new(name: 'card__generic', data: { card_type: 'small', heading: card_heading(laying), list_description: card_list(laying) }).to_h
       end
     end
 
     def card_heading(laying)
       ComponentSerializer::HeadingComponentSerializer.new(content: laying.laid_thing.try(:laidThingName), size: 2, link: statutory_instrument_path(laying.laid_thing.graph_id)).to_h
     end
 
-    def card_paragraph(laying)
-      ComponentSerializer::ParagraphComponentSerializer.new(content: [{ content: "Laying date: #{l(laying.date)}" }]).to_h
+    def card_list(laying)
+      ComponentSerializer::ListDescriptionComponentSerializer.new(items: [{ 'term': { 'content': 'laid-thing.laid-date' }, 'description': [{ 'content': l(laying.date) }] }]).to_h
```

Comment:
this line would be clearer if it was split across multiple lines

Reply:
🎉